From d420557113ee18c9d81e27cfe3aee9ac25b7b13b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Sun, 15 Mar 2020 23:07:07 -0700
Subject: [PATCH 1276/1852] dt: bindings: media: xilinx: sdirxss : Add bit
 depth optional property

commit 8797d6dfa850258449e962b1706b781b6f920534 from
https://github.com/Xilinx/linux-xlnx.git

Add an optional property "xlnx,bpp" which denotes the bit depth
configured in the IP core. The default value is 10.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
index 114e228ae180..169338ed086d 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
@@ -39,6 +39,8 @@ Optional properties:
 
 - reset_gt-gpios: contains GPIO reset phandle for FMC init done pin in GT.
   This pin is active low.
+- xlnx,bpp: This denotes the bit depth as 10 or 12 based on IP configuration.
+  The default value is 10 for backward compatibility.
 
 Example:
 		v_smpte_uhdsdi_rx_ss: v_smpte_uhdsdi_rx_ss@80000000 {
-- 
2.31.1

